griptape/drivers/vector/mongodb_vector_store_driver.py

```diff
@@ -81,10 +81,38 @@ def load_entries(
 
     def query(
         self,
-        query: str,
+        vector: list[float],
         count: Optional[int] = None,
         namespace: Optional[str] = None,
         include_vectors: bool = False,
+        skip: Optional[int] = 0,
         **kwargs
     ) -> list[BaseVectorStoreDriver.QueryResult]:
-        raise NotImplementedError("Vector search not supported in MongoDB")
+        collection = self.get_collection()
+
+        knn_k = skip + (count if count else 10)
+        pipeline = [
+            {
+                "$search": {
+                    "knnBeta": {
+                        "vector": vector,
+                        "path": "vector",
+                        "k": knn_k
+                    }
+                }
+            },
+            {"$skip": skip},
+        ]
+
+        if count is not None:
+            pipeline.append({"$limit": count})
```

Comment:
Based on `(count if count else 10)` above, I think this should also default to 10, right? I'd just set `knn_k` to `count if count else 10` and then set `"k"` to `knn_k + skip` on line 100 and use `knn_k` as a `$limit` value here, so it's less error-prone.